src/main/kotlin/com/github/erosb/jsonsKema/SchemaLoader.kt

```diff
@@ -204,7 +206,7 @@ class SchemaLoader(
                         ?.toList()
                     } ?: emptyList()
             }
-            else -> TODO()
+            else -> throw SchemaLoadingException("unexpected schema : $schemaJson")
```

Comment:
Hello, I would suggest using the already existing `JsonTypingException` like
`else -> throw JsonTypingException("boolean or object", schemaJson.jsonTypeAsString(), schemaJson.location)`

This way the thrown exception would be consistent with the `IJsonValue#require*()` methods and would also include source location. WDYT?

Reply:
Thanks, sounds good!  I changed to `JsonTypingException` as you suggested.